addons/google_gmail/models/ir_mail_server.py

```diff
@@ -12,6 +13,12 @@ class IrMailServer(models.Model):
     _name = 'ir.mail_server'
     _inherit = ['ir.mail_server', 'google.gmail.mixin']
 
+    @api.constrains('use_google_gmail_service')
+    def _check_use_google_gmail_service(self):
+        if self.filtered(lambda server: server.use_google_gmail_service and not server.smtp_user):
+            raise UserError(_(
+                            'Please fill the "Username" field with your Gmail username (your email address). '
+                            'This should be the same account as the one used for the Gmail OAuthentication Token.'))
```

Comment:
Okay can't get it right with the github. For the final commit (merge and squash), could you add a newline here to correctly separate the two definitions?

Reply:
hi @jorv-odoo ,
squashed and updated, can you tell me how to change the author in the commit message, if you know how to change, feel free to change :)